spec/Section 4 -- Composition.md

```diff
@@ -115,6 +115,656 @@ type User {
 scalar Tag
 ```
 
+### Root Mutation Used
+
+**Error Code**
+
+`ROOT_MUTATION_USED`
+
+**Severity**
+
+ERROR
+
+**Formal Specification**
+
+- Let {schemas} be the set of all source schemas.
+- For each {schema} in {schemas}:
+  - Let {rootMutation} be the root mutation type defined in {schema}, if it
+    exists.
+  - Let {namedMutationType} be the type with the name `Mutation` in {schema}, if
+    it exists.
+  - If {rootMutation} is defined:
+    - {rootMutation} must be named `Mutation`.
+  - Otherwise, {namedMutationType} must not be defined.
+
+**Explanatory Text**
+
+This rule enforces that, for any subgraph schema, if a root mutation type is
+defined, it must be named `Mutation`. Defining a root mutation type with a name
+other than `Mutation` or using a differently named type alongside a type
+explicitly named `Mutation` creates inconsistencies in schema design and
+violates the composite schema specification.
+
+Valid Example:
+
+```graphql example
+schema {
+  mutation: Mutation
+}
+
+type Mutation {
+  createProduct(name: String): Product
+}
+
+type Product {
+  id: ID!
+  name: String
+}
```

Comment:
I think without the object type the mutation does not really make any sense

Reply:
To clarify, I meant that it could just be a simple mutation that returns a scalar, but on second thought, even though that's terser, it's probably less realistic.